src/main/java/org/folio/rest/impl/UserRepository.java

```diff
@@ -19,45 +22,74 @@ public UserRepository(VertxOkapiHttpClient client) {
   }
 
   public CompletableFuture<JsonObject> getUser(String userId, Map<String, String> okapiHeaders) {
+    logger.info("getUser::Retrieving user with ID: {}", userId);
     return client.get("/users/" + userId, Map.of(), okapiHeaders)
-      .thenApply(ResponseInterpreter::verifyAndExtractBody);
+      .thenApply(response -> {
+        logger.info("getUser::Successfully retrieved user with ID: {}", userId);
+        return ResponseInterpreter.verifyAndExtractBody(response);
+      });
   }
+
   public CompletableFuture<JsonObject> getUserByEmail(String email, Map<String, String> okapiHeaders) {
+    logger.info("getUserByEmail::Retrieving user by email: {}", email);
```

Comment:
Again, not sure if logging email is a good idea

Reply:
fixed